chalicelib/lib/slack.py

```diff
@@ -136,6 +136,23 @@ def slack_responder(url, msg):
     return res.status_code
 
 
+def slack_message(token, msg, channel):
```

Comment:
Can't we use existing method for this?
https://github.com/codelabsab/timereport-slack/blob/71935ecfcb12c9194a4c985c8277d399ad099942/chalicelib/lib/slack.py#L25

Reply:
Thanks, must have missed that one :)